### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by Rockyraven**
update carbon code and refactor css structured HTML

### CODE REVIEW

File: style/comp-rest.css [hunk 1]
```diff
@@ -35,6 +35,8 @@ main{
     height: auto;
     margin-bottom: 2rem;
     box-shadow: rgba(14, 30, 37, 0.062) 0px 2px 4px 0px, rgba(14, 30, 37, 0.199) 0px 2px 16px 0px;
+    display: flex;
+    flex-wrap: wrap;
 }
```
Comment (Shubhamkr2610) [inline style/comp-rest.css:40]: use color variable name

File: style/comp-rest.css [hunk 2]
```diff
@@ -35,6 +35,8 @@ main{
...
 .code-preview:hover{
     box-shadow: rgba(14, 30, 37, 0.315) 0px 2px 4px 0px, rgba(14, 30, 37, 0.329) 0px 2px 16px 0px;
```
Comment (Shubhamkr2610) [inline style/comp-rest.css:42]: use color variable name

File: style/comp-rest.css (hunk 1 above)
Comment (Rockyraven) [reply style/comp-rest.css:40]: this is only only for box shadow

File: style/comp-rest.css (hunk 2 above)
Comment (Rockyraven) [reply style/comp-rest.css:42]: this is only only for box shadow

**Review by Shubhamkr2610 — APPROVED** [review]
you are good to go

File: component/avatar/avatar.css [hunk 3]
```diff
@@ -1,10 +1,3 @@
-@import url('/style.css');
- @import url('/component/slidebaar.css');
- @import url('/component/alert/alert.css');
-*{
-    margin: 0px;
-    padding: 0px;
-}
 .Extra-large-img img{
```
Comment (vishalg8454) [inline component/avatar/avatar.css]: class naming should start with small letter.

**Review by vishalg8454 — APPROVED** [review]

**Review by arjun-computer-geek — APPROVED** [review]
yay it good to do merge

**Comment by sagargse** [comment]
looks all good to me

File: component/avatar/avatar.css (hunk 3 above)
Comment (Rockyraven) [reply component/avatar/avatar.css]: I have editted class name

**Pull request merged**